.github/workflows/cve-scanning-python.yml

```diff
@@ -0,0 +1,35 @@
+name: CVE Scanning for Python
+
+on:
+  pull_request:
+    paths:
+      - 'python/pyproject.toml'
+      - 'python/safety-policy.yml'
+      - '.github/workflows/cve-scanning-python.yml'
+
+env:
+  PYTHON_KEYRING_BACKEND: keyring.backends.null.Keyring
+
+jobs:
+  scan:
+    name: Build and test App
```

Comment:
Do we need this in cve-scanning  ? We already have the build/test in https://github.com/finos/symphony-bdk-python/blob/main/.github/workflows/build.yml

Reply:
Yes, i believe that it is necessary, since the scanning will occur on the libraries installed in the environment.